contributing.md

```diff
@@ -0,0 +1,18 @@
+# How to contribute to Scoreboard
+
+## Steps to contributing
+
+- Fork the repository
+- Create a branch for your changes
+- Make the changes you'd like to contribute. See the "types of contributions" list above to learn more about what we're looking for
```

Comment:
What are the types of contribution?

Reply:
@kamicut this was copied from a similar file, I'll delete for now.